internal/uvm/create_wcow.go

```diff
@@ -184,7 +184,7 @@ func CreateWCOW(opts *OptionsWCOW) (_ *UtilityVM, err error) {
 
 	hcsSystem, err := hcs.CreateComputeSystem(uvm.id, fullDoc)
 	if err != nil {
-		logrus.Debugln("failed to create UVM: ", err)
+		logrus.WithError(err).Error("failed to create UVM")
```

Comment:
Log level changed from Debug to error here, intentional?

Reply:
I guess not. Probably this is an unnecessary redundant log message. I'll leave it as Debug.